massa-execution-worker/src/interface_impl.rs

```diff
@@ -534,6 +534,10 @@ impl Interface for InterfaceImpl {
         Ok(addr.to_string())
     }
 
+    fn validate_address(&self, address: &str) -> Result<bool> {
```

Comment:
Any reason to return here a Result as a bool looks enough?

Reply:
Indeed, was just mimicking the other functions, but can take it out

Reply:
No I can't, because in the trait definition in `src/types.rs`, it fails as apparently `unimplemented!` will type check as `Result` for clippy